SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -477,6 +479,9 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	check(NewPackageMap == PackageMap);
 
 	PackageMap->Init(this, &TimerManager);
+
+	// Pass the PackageMap to the InterestFactory. The PackageMap was not available to pass when the InterestFactory was created.
```

Comment:
This is a little sketchy, we're using the InterestFactory `CreateClientCheckoutRadiusConstraint` before we've fully set it up. Does the interest factory need the package map?

Reply:
We use the package map at SpatialInterestFactory::AddObjectToConstraint which looks like it is used to add EntityId constraints. If this is a problem I can make CreateClientCheckoutRadiusConstraint  static and pass in the ClassInfoManager. I could then move InterestFactory creation near the package map setup.

Reply:
What do we want to do here?